.eslintignore

```diff
@@ -3,3 +3,9 @@
 **/staticBundle.js
 **/main.js
 Libraries/vendor/**/*
+RNTester/**/*
```

Comment:
But we do write some, so it would be good if we linted it... just add a backlog issue to re-enable linting for RNTester